cfgov/jinja2/v1/_includes/molecules/contact-address.html

```diff
@@ -24,11 +22,14 @@
 {% macro render(address) %}
 <div class="m-contact-address">
     {% set icon = 'cf-icon-mail' %}
-    {% set title = address.label %}
+    {% set label = address.label %}
```

Comment:
Hm, it was originally there for consistency with `contact-phone`, but yeah it's not necessary in `contact-address` and `contact-email`. Fixed in https://github.com/cfpb/cfgov-refresh/commit/6b0d6a8e6bc13dfa3ca7579d861109431ef24f57